views/peeps/new.erb

```diff
@@ -0,0 +1,7 @@
+<h1>New peep</h1>
+
+<form action="/peeps" method="post">
+  <input type="text" name="message" placeholder="Message"/>
+  <input type="hidden" name="date" value="<%=Date.today%>">
```

Comment:
Using a hidden field works but it does mean that a malicious user could issue their own POST request and set a time of their choosing. The other pitfall is that the date will be generated at the time when this page is rendered rather than at the time when the user clicks submit, so at midnight boundaries it might end up being wrong.

For reasons like that it can be better to generate the date in your Model code just before you insert the data into the database.